services/s3/src/main/java/software/amazon/awssdk/services/s3/internal/handlers/DecodeUrlEncodedResponseInterceptor.java

```diff
@@ -17,16 +17,22 @@
 
 import static software.amazon.awssdk.utils.http.SdkHttpUtils.urlDecode;
 
+import java.util.Collections;
```

Comment:
Should consider adding a backlog task to automate at least catching future S3 operations that expose similar behavior?

We could probably make it as simple as operations that have an `encodingType` parameter

Reply:
+1